### DISCUSSION TITLE
How do I configure the timeout for a PostgREST operation?

### DISCUSSION THREAD

**Discussion by pedrommcarrasco** [supabase/supabase#29300] [Questions] [answered]
I couldn’t find this information in the documentation or API. Is there a way to configure the timeout for an operation like the one below using the Swift SDK?

```swift
try await networkApi
    .from(SyncCommandDatabaseRecord.databaseTableName)
    .insert(record)
    .execute()
```

**Comment by silentworks** [comment, marked as answer]
This documentation page says there is a set timeout of 60 seconds for API calls. If you need to make longer calls you should use direct connection. Read more in these docs https://supabase.com/docs/guides/database/postgres/timeouts

**Reply by pedrommcarrasco** [reply]
Thank you!
